cli_helper/kart.c

```diff
@@ -209,6 +209,17 @@ void exit_on_alarm(int sig)
     exit(exit_code);
 }
 
+/**
+ * @brief Exit signal handler for SIGINT.
+ * Tries to kill the whole process group.
+ */
+void exit_on_sigint(int sig)
+{
+    putchar('\n');
```

Comment:
what's the putchar for?

Reply:
Hitting Ctrl+C often ends up with the last line of output and your PS1 prompt on the same line / mixed together. No other program seems to care about this but I like to LF out of the way